DOL.WHD.Section14c.Web/src/modules/components/accountManagementControls/accountFormDirective.test.js

```diff
@@ -0,0 +1,26 @@
+describe('accountForm', function() {
+
+    beforeEach(module('14c'));
+
+    var element, rootScope;
+    beforeEach(function () {
+        element = angular.element('<account-form/>');
+    	inject(function ($rootScope, $compile, _$q_, apiService) {
+            $q = _$q_;
+            rootScope = $rootScope;
+            mockApiService = apiService;
+
+            getAccount = $q.defer();
+            spyOn(mockApiService, 'getAccount').and.returnValue(getAccount.promise);
+
+            getRoles = $q.defer();
+            spyOn(mockApiService, 'getRoles').and.returnValue(getRoles.promise);
+
+            $compile(element)(rootScope);
+        });
+    });
+
+    it('invoke directive', function() {
+        rootScope.$digest();
```

Comment:
updated directive unit tests to assert it was created https://github.com/AppliedIS/dol-whd-14c/commit/e73d8d8cc8291dbbcf47a7da435a7ad963cc54a4